packages/terra-clinical-site/src/examples/alert/LongTextExample.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Alert from 'terra-clinical-alert';
+import Button from 'terra-button';
+import Toggle from 'terra-toggle';
+
+class LongTextExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      allTextShown: false,
+    };
+    this.handleShowMore = this.handleShowMore.bind(this);
+    this.handleShowLess = this.handleShowLess.bind(this);
+  }
+
+  handleShowMore() {
+    this.setState({ allTextShown: true });
+  }
+
+  handleShowLess() {
+    this.setState({ allTextShown: false });
+  }
+
+  render() {
+    const alertStartTagStr = '<Alert type={Alert.Opts.Types.INFO} title="Gettysburg Address:" >';
+    const alertContentStr = '<span>Four score and seven years ago our fathers brought forth on this continent, a new nation, . . .</span>';
+    const alertEndTagStr = '</Alert>';
+    return (
+      <div>
+        <div dir="ltr">
+          <h3>Alert of type info with custom title and long text with show more/less</h3>
+          <br />
+          <code>
+            {alertStartTagStr}
+            <br />
+            &nbsp;&nbsp;{alertContentStr}
+            <br />
+            {alertEndTagStr}
+          </code>
+        </div>
+        <br />
+        <Alert type={Alert.Opts.Types.INFO} title="Gettysburg Address:" >
+          <span>
+            Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.
+            <Toggle isOpen={!this.state.allTextShown} isAnimated>
+              <Button onClick={this.handleShowMore} text="Show More" variant="link" size="medium" />
+            </Toggle>
+            <Toggle isOpen={this.state.allTextShown} isAnimated>
```

Comment:
@jmsv6d Initial lookover looks good. Let's update the "Alert of type info with custom title and long text with show more/less" example to not be isAnimated for the toggle.